schemes/rrtmgp/rrtmgp_sw_cloud_optics.F90

```diff
@@ -1,5 +1,12 @@
+!> \file rrtmgp_sw_cloud_optics.F90
+!!
 module rrtmgp_sw_cloud_optics
-use ccpp_kinds, only: kind_phys
+  use ccpp_kinds, only: kind_phys
+  use rrtmgp_cloud_optics_setup, only: g_lambda, g_mu, nmu, nlambda
+  use rrtmgp_cloud_optics_setup, only: g_d_eff, n_g_d
+  use rrtmgp_cloud_optics_setup, only: ext_sw_liq, ext_sw_ice
+  use rrtmgp_cloud_optics_setup, only: asm_sw_liq, asm_sw_ice
+  use rrtmgp_cloud_optics_setup, only: ssa_sw_liq, ssa_sw_ice
```

Comment:
It looks like many of these `use` statements already exist at the subroutine level.  So can we delete the ones here that are already in the subroutine, and then move the remaining ones down to the subroutine scope?  That would better match CAM-SIMA coding standards.

Reply:
moved!